lib/orbit/import/import_helpers.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Orbit.Import.ImportHelpers do
```

Comment:
This file should probably have tests. It'd mostly be exercised by the tests of the individual importers, but those cases would be complex, so it'd be nice to have some simple tests for these functions, too.

Reply:
yeah, you're right. here are a couple for `parse_csv`—the other two functions are really just wrappers? https://github.com/mbta/orbit/pull/203/files#diff-b5202e24be21249b1155432d249572dd4a4b0a3b844f445fe4051215d31579deR1